site/content/concepts/identifier-use/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<!--
+    Some older versions of OSCAL refer to the identifier use page with a relative path.
+
+    This alias ensures no links will break even for those older versions.
+-->
```

Comment:
NB: I am fine with this, shouldn't we just fix the aliases in OSCAL-Pages as well?

Reply:
The aliases are fixed there too (via https://github.com/usnistgov/OSCAL-Pages/pull/51). This fix is referring to older source metaschemas that refer to pages with relative links (such as a remark directly to `/concepts/identifier-use/`, not a URL).

Reply:
That makes sense. I approved, but I will resolve this anyway.